README.md

```diff
@@ -989,6 +989,92 @@ when the content is present.
   end
 ```
 
+## Custom Components
+
+When you use custom wrappers, you might also be looking for a way to add custom components to your 
+wrapper. The default components are:
+
+```ruby
+:label         # The <label> tag alone
+:input         # The <input> tag alone
+:label_input   # The <label> and the <input> tags
+:hint          # The hint for the input
+:error         # The error for the input
+```
+
+A custom component might be interesting for you if your views look something like this:
+
+```erb
+<%= simple_form_for @blog do |f| %>
+  <div class="row">
+    <div class="span1 number">
+      1
+    </div>
+    <div class="span8">
+      <%= f.input :title %>
+    </div>
+  </div>
+  <div class="row">
+    <div class="span1 number">
+      2
+    </div>
+    <div class="span8">
+      <%= f.input :body, as: :text %>
+    </div>
+  </div>
+<% end %>
+```
+
+A cleaner method to create your views would be:
+
+```erb
+<%= simple_form_for @blog, wrapper: :with_numbers do |f| %>
+  <%= f.input :title, number: 1 %>
+  <%= f.input :body, as: :text, number: 2 %>
+<% end %>
+```
+
+To use the number option on the input, first, tells to Simple Form the place where the components 
+will be:
+
+``` ruby
+# config/initializers/simple_form.rb
+Dir[Rails.root.join('app/components/**/*.rb')].each { |f| require f }
```

Comment:
`app/components` will be in a eager load path, do we need to require? Also it will be loaded at boot what may cause autoload problems. Maybe we want to put them in lib to avoid gotchas with autoload?

Reply:
Even inside `app` folder we need to require the file, without that it doesn't work.

I don't know if it will cause autoload problems, do you think it's better put the components inside `lib/components/`?

Reply:
@feliperenan If you need a reference implementation, I've granted you access to [m5o/simple_form-bootstrap](https://github.com/m5o/simple_form-bootstrap) this is working example.

Reply:
Yes. `lib` is the correct place for it.

Reply:
@rafaelfranca Done!. Do you think we are ready to go ahead?